Visual Studio/Realm.vcxproj

```diff
@@ -167,13 +167,10 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Static library, release|x64'" Label="Configuration">
     <ConfigurationType>StaticLibrary</ConfigurationType>
     <CharacterSet>Unicode</CharacterSet>
-    <AppContainerApplication>false</AppContainerApplication>
-    <ApplicationType>Windows</ApplicationType>
-    <ApplicationTypeRevision>10.0</ApplicationTypeRevision>
+    <WholeProgramOptimization>
+    </WholeProgramOptimization>
     <TargetPlatformVersion>8.1</TargetPlatformVersion>
     <WindowsAppContainer>false</WindowsAppContainer>
-    <WholeProgramOptimization>false</WholeProgramOptimization>

```

Comment:
Isn't `WholeProgramOptimization` off by default, though? I just checked and I don't see the `-CL` switch being passed to `cl.exe` when building.

I just copied the properties from the win32 platform and this came along.